src/database/firebase_object_factory.spec.ts

```diff
@@ -81,12 +81,12 @@ describe('FirebaseObjectFactory', () => {
 
 
     it('should emit unwrapped data by default', (done: any) => {
-      ref.set({ data: 'bar' }, () => {
+      const fixtureData = { data: 'bar' };
+      ref.set(fixtureData, () => {
         subscription = observable.subscribe(unwrapped => {
```

Comment:
Sorry, my brain was malfunctioning. I looked at this too early in the morning. I forgot that there's a `FirebaseObjectObservable` variable in the outer scope. All is good!